icaruscode/CRT/CRTT0MatchingAna_module.cc

```diff
@@ -338,7 +338,7 @@ namespace icarus {
 	  //std::cout << "closest match " << closest.t0 << std::endl;
 	  //std::cout << "closest dca " << closest.dca << " ,tagger: "<< closest.thishit.tagger << " , sin angele: \t" << closest.dca/closest.extrapLen << std::endl;
 	  double sin_angle = -99999;
-	  if(closest.dca != -99999){ 
+	  if(closest.dca != DBL_MIN){ 
```

Comment:
In other parts of the code (e.g. [`CRTT0MatchAlg::GetClosestCRTHit()`](https://github.com/SBNSoftware/icaruscode/blob/c80d7593f248a8da615c454445076ff790cba89d/icaruscode/CRT/CRTUtils/CRTT0MatchAlg.cc#L556), [`CRTT0MatchAlg::T0AndDCAFromCRTHits()`](https://github.com/SBNSoftware/icaruscode/blob/c80d7593f248a8da615c454445076ff790cba89d/icaruscode/CRT/CRTUtils/CRTT0MatchAlg.cc#L654)) this type of check appears to me to be expressed as negativity check:
```suggestion
	  if(closest.dca >= 0.0){ 
```
It may be more robust than relying on a specific magic value.